### PR TITLE
Move preload indicator css to own file

### PR DESCRIPTION
**Pull request by eatonphil**

### CODE REVIEW

File: scss/components/index.scss [hunk 1]
```diff
@@ -3,8 +3,8 @@
 @import 'datacenter';
 @import 'form';
 @import 'backup';
-@import 'status-dropdown';
```
Comment (briansteffens) [inline scss/components/index.scss]: Why was this removed in this PR?

Comment (eatonphil) [inline scss/components/index.scss]: It's in the wrong place

**Comment by eatonphil** [comment]
thanks!
